src/test/integration/integrationtests.py

```diff
@@ -6,9 +6,37 @@
 
 class gateway_tests(unittest.TestCase):
 
-	# Tests that a good request returns a 200
-	def test_success(self):
-		self.assertEqual(good_request(url, access_token), 200)
+	# Tests a single resource ID in different case styles
+	def test_id(self):
```

Comment:
Can you update these python files so that they are using spaces: https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces ? Also can you set the default in your editor / IDE to be spaces?